README.md

```diff
@@ -130,45 +132,104 @@ deploy_heroku(BENTO_BUNDLE_PATH, DEPLOYMENT_NAME, CONFIG_JSON)
 Use command line
 
 ```bash
-python update.py <Bento_bundle_path> <Deployment_name> <Config_JSON>
+$ ./update <Bento_bundle_path> <Deployment_name> <Config_JSON>
 ```
 
 Use Python API
 
 ```python
-from update import update_heroku
+from heroku_deploy import update
 
-update_heroku(BENTO_BUNDLE_PATH, DEPLOYMENT_NAME, CONFIG_JSON)
+update(BENTO_BUNDLE_PATH, DEPLOYMENT_NAME, HEROKU_CONFIG)
 ```
+* where `HEROKU_CONFIG` is a dictionary with keys for `"dyno_counts"` and `"dyno_type"`
 
 ### Get a deployment's status and information
 
 Use command line
 
 ```bash
-python describe.py <Deployment_name>
+$ ./describe <Deployment_name>
 ```
 
 Use Python API
 
 ```python
-from describe import describe_heroku
+from heroku_deploy import describe
 
-describe_heroku(DEPLOYMENT_NAME)
+describe(DEPLOYMENT_NAME)
 ```
 
 ### Delete a deployment
 
 Use command line
 
 ```bash
-python delete.py <Deployment_name>
+$ ./delete <Deployment_name>
 ```
 
 Use Python API
 
 ```python
-from delete import delete_heroku
+from heroku_deploy import delete
 
-delete_heroku(DEPLOYMENT_NAME)
+delete(DEPLOYMENT_NAME)
+```
+
+## Operator Deployment
+
+To add the Heroku Deployment Tool as an operator for the Bento Cloud Deployment Tool:
+
+1. Install `bcdt` from the [repo](https://github.com/bentoml/cloud-deployment-tool/tree/prototype)
```

Comment:
This is great instruction to get thing moving.  Let's make it for the real release instead of the current prototype. Maybe `1. Install `bcdt` with pip ```pip install bcd``` or Install `bentoctl` with pip ```pip install bentoctl````

Reply:
Updated. Will make sure to update all README's with the new name once it's been finalized